committee-steering/governance/annual-reports.md

```diff
@@ -26,56 +26,29 @@ to make this an agenda item of topic to discuss together. Chairs and Organizers
-  - share with their groups (see tips below) and begin to 
-compile the doc due April 8th - final version
-  - use this [question template]
-  - submit a pull request with at least the template questions against the set 
-  issue to the groups documentation in the community repo by March 8th
-  - verify that Steering Liaison has reviewed
-  - post a call for comments period with your groups mailing list and a complete 
-  questionnaire; you can follow your charter communication period if you have
-  one but no less than 72 hours 
-  - if there are private matters to address with Steering before submitting the 
-pull request, share a copy with your liaison and wait until comments are
-addressed before submitting.
-3. The March edition of the "Chairs, Tech Leads, and Organizers" meetings will 
-be used as follow up for the community groups that have questions from/to 
-Steering and a retrospective
-4. The Steering Committee liaison will work directly with groups that have 
-follow up items and update Steering during regular monthly meetings. The liaison
-will also coordinate time with the Chairs (as a group). If you'd like to meet 
-1:1 instead, please let your liaison know.
-We'd like to hear about the following:  
-  - goals you have for the group, your experience in the role so far - 
-  challenges, wins, things you didn’t know before but wish you did, 
-  - continue in the role or help finding a replacement if you've already asked 
-  the group, 
-  - any feedback you have for us as a body and how we can help Chairs and 
-Organizers succeed and feel comfortable in your role. 
-
-### 2020:
-Due to playing catch up with 37 community groups, we will have a slightly
-modified timeline bootstrap process this year and then will follow the process
-outlined for 2021. Working Groups, due to their temporary nature, will be the 
-only group to submit for 2020.   
-
-1. Steering Committee will assign one member as a liaison to each community 
-group for kick off, help, or guidance needed throughout the process during the 
-first week of July
-2. Working Group Organizers create one report and send it to/share it with 
-steering-private@kubernetes.io for review; reports should take ~30 days to 
-complete (work with your liaison on timeline)
-3. Steering will schedule private meetings to follow up on anything denoted as
-a private matter on the report; reports are not made public until all private
-comments have been addressed
-4. Once reviewed and follow ups are completed, the liaison will ask the 
-organizer(s) of the group to submit a pull request to that groups documentation 
-in the community repo
-5. The Steering Committee liaison will work directly with groups that have 
-follow up items and update Steering during regular monthly meetings. 
+1. Early January (of the following year)
+   * Steering Committee finalizes [questions] and generates draft 
+     `annual-report-YYYY.md` templates for each group in the community repo
```

Comment:
@cblecker does this match your expectations?

Reply:
Yup!